psqlpy-0.3.5.dist-info/METADATA,sha256=PGbGivMNd7SlZEH2rTmPQxJaxfAEi1g159CbuREb9JI,18466
psqlpy-0.3.5.dist-info/WHEEL,sha256=5uRTlaeR8a9TbM3Ic9HydYdleo8HIZHpWpS4jHsD38g,131
psqlpy-0.3.5.dist-info/license_files/LICENSE,sha256=UkxvzBjZdhB37ez6jaooRIOe3GON6A2MJ7RVsiOxlUc,1078
psqlpy/__init__.py,sha256=0xm7091LiOrM0kZmJZ1C7DLLTNCOXEbFLUb_p0w11Tk,334
psqlpy/extra_types.py,sha256=mNGroV2GC1gwGM1dhzyRNt33eet4EEv7MwDget6LNHg,258
psqlpy/py.typed,sha256=47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU,0
psqlpy/_internal/exceptions.pyi,sha256=8n0Mo6GzRY8AcQftxMhYCMIWnIpjzYQ7K1DGDlJi-oo,1315
psqlpy/_internal/extra_types.pyi,sha256=6cOrOkYz2Y_7g8TduLpCw8SwSOdCWmZCdTtNYrxXxAU,2003
psqlpy/_internal/__init__.pyi,sha256=2KHITORzsJmpJxGyLX3XLpjLolLOlCTEIDDXPWkXd7s,28351
psqlpy/exceptions.py,sha256=xNKilC9h-LHVkpCAOfWkDPhoVIcbVgRD5yBDDZ2aivo,511
psqlpy/_internal.cpython-312-aarch64-linux-gnu.so,sha256=Q6JJ_r8hVN0zY7ooFG7vwuqyDNgFQPX8U8e1cBGt_Vg,9105112
psqlpy-0.3.5.dist-info/RECORD,,
